### PR TITLE
added updateItemsOrdination mutation

### PR DESCRIPTION
**Pull request by marcosewbank**
#### What problem is this solving?

This new feature will allow to sort items at `orderForm` by name or date of addition using the new `itemsOrdination` mutation.

#### How should this be manually tested?

1. Go to this [Workspace](https://ecom5991--carrefourbrfood.myvtex.com/) and add some products to the cart

2. Run this [mutation](https://ecom5991--carrefourbrfood.myvtex.com/_v/private/vtex.checkout-graphql@0.44.1/graphiql/v1?operationName=itemsOrdination&query=mutation%20itemsOrdination(%24ascending%3A%20Boolean!%2C%20%24criteria%3A%20String!)%20%7B%0A%20%09itemsOrdination(ascending%3A%20%24ascending%2C%20criteria%3A%20%24criteria)%20%7B%0A%20%09%09id%0A%20%20%20%20items%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20skuName%0A%20%20%20%20%7D%0A%09%7D%0A%7D%20&variables=%7B%0A%20%20%22ascending%22%3A%20true%2C%0A%20%20%22criteria%22%3A%20%22add_time%22%0A%7D)
3. Then reload the page.

Now the products are reordered :smiley: 

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: graphql/types/OrderForm.graphql [hunk 1]
```diff
@@ -66,3 +66,8 @@ input ClientPreferencesDataInput {
   optInNewsletter: Boolean
   locale: String
 }
+
+enum ItemsOrdinationCriteria {
+  name,
```
Comment (lucasecdb) [inline graphql/types/OrderForm.graphql:71]:
```suggestion
  name
```

File: graphql/schema.graphql [hunk 2]
```diff
@@ -81,4 +81,9 @@ type Mutation {
   setManualPrice(orderFormId: ID, input: ManualPriceInput!): OrderForm!
     @withOrderFormId
     @cacheControl(scope: PRIVATE)
-}
+
+  updateItemsOrdination(orderFormId: ID, ascending: Boolean!, criteria: ItemsOrdinationCriteria!): OrderForm!
+    @withOrderFormId
+    @cacheControl(scope: PRIVATE)
+
+}
```
Comment (lucasecdb) [inline graphql/schema.graphql:89]: can you add the newline at the end of the file again?

File: node/clients/checkout.ts [hunk 3]
```diff
@@ -402,6 +411,7 @@ export class Checkout extends JanusClient {
       savePaymentToken: (queryString: string) =>
         `${base}/current-user/payment-tokens/${queryString}`,
       getPaymentSession: () => `${base}/payment-session`,
+      updateItemsOrdination: (orderFormId: string) => `${base}/orderForm/${orderFormId}/itemsOrdination`
     }
   }
-}
+}
```
Comment (lucasecdb) [inline node/clients/checkout.ts:417]: same, please add the newline again

File: node/resolvers/orderForm.ts [hunk 4]
```diff
@@ -259,4 +259,26 @@ export const mutations = {
...
+      
+    const changeItemsOrdination = await checkout.updateItemsOrdination(
+      orderFormId!,
+      ascending, 
+      criteria
+    )
+
+    return changeItemsOrdination
+  },
+}
```
Comment (lucasecdb) [inline node/resolvers/orderForm.ts:284]: same as above

**Review by lucasecdb — APPROVED** [review]

**Review by alvimm — APPROVED** [review]

**Pull request merged**